### DISCUSSION TITLE
[question] how to disable this message?

### DISCUSSION THREAD

**Discussion by fengscarleteyes** [folke/snacks.nvim#229] [Q&A]
- when i use `Snacks.scratch()` & `Snacks.dashboard.open()` ...
- will show a message [ picture ]
- how to disable this message?

---

- I am not very familiar with the translator used in English. My attitude is to sincerely seek advice and hope that the translator's vocabulary can be expressed correctly
  - 我不太懂英语 使用的翻译器 我的态度是 诚恳的请教 希望翻译器的用词 能正确表达

---

![1 1](https://github.com/user-attachments/assets/11878ef7-71de-4d96-8d86-0e3bf282b179)
![2 2](https://github.com/user-attachments/assets/56e468fa-681e-4717-a11a-c4472fcd4b64)

**Comment by folke** [comment]
That's because you do `:= Snacks...`. You're printing the result of that call.
Use `:lua Snacks...` instead

**Reply by fengscarleteyes** [reply]
thank you!
